frontend/public/js/autopresets.js

```diff
@@ -1,13 +1,62 @@
+var columns = 3;
```

Comment:
I added some comments and cleaned up the script a bit in 14cff5cb6f30340b61d71f36b64960a5618ef33f